tasks/main.yml

```diff
@@ -102,8 +120,8 @@
     follow: False
 
 - name: Check if resolvconf service exists
-  command: systemctl list-unit-files resolvconf.service
-  failed_when: False
+  command: systemctl cat resolvconf.service
+  failed_when: resolvconf_service_list.rc not in [0, 1]
```

Comment:
Why this change?

Reply:
It's been a while, so I may not remember the reason correctly, but I believe the former approach didn't set the return code of the command on some systems, so the following test on line 134 will not work.
Just tested it:
on debian 11.2:
```
$ systemctl list-unit-files nonexistent.service > /dev/null 2>&1 ; echo $?
1
$ systemctl cat nonexistent.service > /dev/null 2>&1 ; echo $?
1
```
on debian 10.13:
```
$ systemctl list-unit-files nonexistent.service > /dev/null 2>&1 ; echo $?
0
$ systemctl cat nonexistent.service > /dev/null 2>&1 ; echo $?
1
```